src/plugins/blocksmith/components/image-block-modal/styles.scss

```diff
@@ -0,0 +1,4 @@
+.custom-options-container {
```

Comment:
@jalopez13 not looking to move the styles, wondering why they are needed at all.  bootstrap has a lot of built-ins and admin-ui has components/options that should make this fixed css un-needed.

Reply:
@gdbrown 

It's due to the container div. If that styling is not applied it stretches 100% width of the modal window. Giving it a width of 350px and margin: 0 auto to center it is needed.

Reply:
@jalopez13 - can we apply styles to the element directly? see my comment above.